stable/kommander/README.md

```diff
@@ -0,0 +1,29 @@
+# Kommander
+Represents an umbrella chart for deploying multiple charts:
+- Kommander-Karma
+- Kommander-Thanos
+- Grafana
+- Kommander-Cluster-Lifecycle
+- Kommander-ui
+- Kubeaddons-Catalog
+
+## Pre-Reqs for Installation
+You must have installed in your cluster `Cert-Manager`.
+
+```bash
+CERT_MANAGER_VERSION=0.10.1
+kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/v${CERT_MANAGER_VERSION}/deploy/manifests/00-crds.yaml;
+helm repo add jetstack https://charts.jetstack.io;
+helm repo update;
+helm install jetstack/cert-manager --name cert-manager --version="${CERT_MANAGER_VERSION}" --namespace cert-manager;
+```
+
+## Install Chart with a known Issuer
+```bash
+helm install --namespace "kommander" --name "kommander-kubeaddons" --set kommander-cluster-lifecycle.certificates.issuer.name="issuer-name" ./stable/kommander
```

Comment:
This review is a little late but I was running through these instructions and got stuck on this step. Here are the steps I took:

1. kind (0.7.0) create cluster (1.17)
2. helm init but with service account/rbac support (https://github.com/helm/helm/blob/240e539cec44e2b746b3541529d41f4ba01e77df/docs/rbac.md#example-service-account-with-cluster-admin-role)

then I followed the instructions here. The error I received after helm install was `Error: timed out waiting for the condition` but I also received this error when doing a helm del --purge on the release. Re-running `helm del` gave me a different error: `Error: jobs.batch "cleanup-kommander-kubeaddons-ops-portal-username" already exists`

Did I neglect some other prereq? Or maybe this was unlucky?